graphql/schema/query.graphql

```diff
@@ -42,11 +42,11 @@ type Query {
   pod(podId: String!): Pod!
   
   # patch
-  patch(patchId: String!): Patch!
+  patch(patchId: String! @requireProjectAccess(permission: TASKS, access: VIEW)): Patch!
 
   # project
-  githubProjectConflicts(projectId: String!): GithubProjectConflicts!
-  project(projectIdentifier: String!): Project!
+  githubProjectConflicts(projectId: String! @requireProjectAccess(permission: SETTINGS, access: VIEW)): GithubProjectConflicts!
+  project(projectIdentifier: String! @requireProjectAccess(permission: TASKS, access: VIEW)): Project!
```

Comment:
should this be checking the settings permission as well?

Reply:
oops, I thought I left a comment on this one, but I can add one. It has to be generally accessible because we surface project banners & patches from this mutation. So people with `basic_project_access` would no longer be able to see project banners or patches if we restricted it to SETTINGS. 😅

Reply:
Ah okay -- for now since we haven't done DEVPROD-3489, I think this might still be something we ticket. I'm fine with going forward here though since variables aren't exposed.

Reply:
Sure, I'll make a ticket and mark it as related 🙌